### PR TITLE
Add multi admin test

### LINKED ISSUE #16431: [MF-1.3]open commissioning window command is failing
Open commissioning window test step is failing,please refer attached logs and screenshot 

Test step:

Testplan :

Error screenshot:

[mf-1.zip](https://github.com/project-chip/connectedhomeip/files/8304521/mf-1.zip)

### PR DESCRIPTION
**Pull request by krypton36**
#### Problem
Test Test_TC_MF_1_3, multi-admin test, does not run correct steps.

#### Change overview
- Update the steps in Test_TC_MF_1_3
- Regenerate code

#### Testing
- Generated Code
- Compiled
- Ran test against chip-all-clusters-app

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 1]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
-          [1635864972.827215][6022:6027] CHIP:TOO: nodeId: 1
-          [1635864972.827251][6022:6027] CHIP:ZCL: Label:
-          [1635864972.827298][6022:6027] CHIP:TOO: FabricDescriptor[2]:
-          [1635864972.827338][6022:6027] CHIP:TOO: fabricIndex: 2
-          [1635864972.827375][6022:6027] CHIP:ZCL: RootPublicKey: 65
-          [1635864972.827440][6022:6027] CHIP:TOO: vendorId: 45896
-          [1635864972.827479][6022:6027] CHIP:TOO: fabricId: 0
-          [1635864972.827517][6022:6027] CHIP:TOO: nodeId: 2"
-      disabled: true
+    #Issue 16094 Need to check length of list and verify its content
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:115]: Again, this is already checking the length....

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 2]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
+    - label: "Query fabrics list"
+      identity: "beta"
+      command: "readAttribute"
+      cluster: "Operational Credentials"
+      attribute: "Fabrics"
+      response:
+          value: [{ Label: "", nodeId: nodeId2 }]
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:122]: We should also test that doing a non-fabric-filtered write returns both fabrics, right?

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 3]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
+          constraints:
+              type: list
 
-    - label: ""
-      verification:
-          "1. Verify if the DUT is broadcasting using $ avahi-browse -rt
-          _matter._tcp"
-      disabled: true
+    - label:
+          "TH_CR1 writes the Basic Information Clusters NodeLabel mandatory
```
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:127]: The [test](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/multiplefabrics.adoc#test-procedure-2) plan indicates to write and read from the Controller 1 again to make sure pairing a second doesn't interfere with the original. I do think we should change the value on each write and read to make sure it is updated.

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 4]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
-          [1635864875.635407][3884:3889] CHIP:TOO: fabricIndex: 2
-          [1635864875.635430][3884:3889] CHIP:ZCL: RootPublicKey: 65
-          [1635864875.635456][3884:3889] CHIP:TOO: vendorId: 45896
-          [1635864875.635480][3884:3889] CHIP:TOO: fabricId: 0
-          [1635864875.635506][3884:3889] CHIP:TOO: nodeId: 2
-          [1635864875.635535][3884:3889] CHIP:ZCL: Label:
-          [1635864875.635633][3884:3889] CHIP:EM: Piggybacking Ack for
-          MessageCounter:1 on exchange: 44071i"
-      disabled: true
+    #Issue 16094 Need to check length of list and verify its content
```
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:105]: Yes it is. Removing commands. Thank you.

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml (hunk 2 above)
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:122]: Actually, the test step does indicate to validate both. Thank you. I apologies for all these inconsistencies. I uploaded the YAML from GRL without reviewing it. Updating.

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 5]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
-          [1635865075.829950][3904:3909] CHIP:ZCL: status: Success (0x0000)"
-      disabled: true
+          "TH_CR2 writes the Basic Information Clusters NodeLabel mandatory
+          attribute of DUT_CE"
+      identity: "beta"
+      command: "writeAttribute"
+      cluster: "Basic"
+      attribute: "NodeLabel"
+      arguments:
+          value: "chiptest"
```
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:155]: Yes!

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 6]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
+          "TH_CR1 writes the Basic Information Clusters NodeLabel mandatory
+          attribute of DUT_CE"
+      command: "writeAttribute"
+      cluster: "Basic"
+      attribute: "NodeLabel"
+      arguments:
+          value: "chiptest"
+
+    - label:
+          "TH_CR1 reads the Basic Information Clusters NodeLabel mandatory
```
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml:136]: Second read is required by test plan. Please see comment above.

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml [hunk 7]
```diff
@@ -16,391 +16,153 @@ name: 22.1.3. [TC-MF-1.3] Node Behavior using ECM [DUT - Commissionee]
...
+          attribute of DUT_CE"
+      command: "readAttribute"
+      cluster: "Basic"
+      attribute: "NodeLabel"
+      response:
+          constraints:
+              type: string
+              maxLength: 32
+              value: "chiptest"
```
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml]: Done

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml]: Removed

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml (hunk 1 above)
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml]: Removed

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml (hunk 2 above)
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml]: Added.

File: src/app/tests/suites/certification/Test_TC_MF_1_3.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (krypton36) [reply src/app/tests/suites/certification/Test_TC_MF_1_3.yaml]: Updated

**Review by bzbarsky-apple — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**